osu.Game/Overlays/BeatmapSet/MetadataSectionSource.cs

```diff
@@ -17,9 +17,9 @@ public MetadataSectionSource(Action<string>? searchAction = null)
         protected override void AddMetadata(string metadata, LinkFlowContainer loaded)
         {
             if (SearchAction != null)
-                loaded.AddLink(metadata, () => SearchAction(metadata));
+                loaded.AddLink(metadata, () => SearchAction($"\"{metadata}\""));
             else
-                loaded.AddLink(metadata, LinkAction.SearchBeatmapSet, metadata);
+                loaded.AddLink(metadata, LinkAction.SearchBeatmapSet, $"\"{metadata}\"");
```

Comment:
Is this going to be enough? See: https://github.com/ppy/osu/blob/2103b3e1861dd83b6fb4f3920bea6d3876cb1691/osu.Game/Overlays/BeatmapSet/BeatmapSetHeaderContent.cs#L245-L262

(yes, that is double quotes)

Reply:
Answer is it's not. Will fix.

Reply:
This is pretty fucked, do we have a solution for this? As a user I don't want to have to double-quote to make search work as expected.

Reply:
As discussed IRL, `{type}=""{text}""` is the format for searching for that specific metadata type, whereas "" is for full-string search.